opencensus/trace/span.h

```diff
@@ -153,7 +168,8 @@ class Span final {
   const SpanContext context_;
 
   // Shared pointer to the underlying Span representation. This is nullptr for
-  // Spans which are not recording events.
+  // Spans which are not recording events. This is an implementation detail, not
+  // part of the public API.
   std::shared_ptr<SpanImpl> span_impl_;
```

Comment:
Why is this not const? Based on the current implementation this can be const. Until we need it to not be please make it const.

Reply:
Mode const.